docs/language/pipeline-model.md

```diff
@@ -19,11 +19,11 @@ refers to a collection of data called a "data pool" and is referenced using
 the pool's name much as SQL references database tables by their name.
 
 For more detail, see the reference page of the [`from` operator](operators/from.md),
-but as an example, you might use the `get` form of `from` to fetch data from an
+but as an example, you might use its `get` form to fetch data from an
 HTTP endpoint and process it with `super`, in this case, to extract the description
 and license of a GitHub repository:
 ```
-super -f text -c 'get https://api.github.com/repos/brimdata/super
+super -f line -c 'get https://api.github.com/repos/brimdata/super
```

Comment:
Mode `line` gives the same output as `text` in this example, so I opted to flip it because we sometimes seem close to dumping `text` as an output format.